apollo-router/src/plugins/progressive_override/tests.rs

```diff
@@ -136,16 +133,10 @@ async fn assert_expected_and_absent_labels_for_supergraph_service(
     .unwrap()
     .supergraph_service(mock_service.boxed());
 
-    // plugin depends on the parsed document being in the context so we'll add
-    // it ourselves for testing purposes
-    let schema = Schema::parse_and_validate(SCHEMA, "").unwrap();
-    let document = Document::parse(query, "query.graphql").unwrap();
-    let executable = document.to_executable(&schema).unwrap();
-    let parsed_doc: ParsedDocument = Arc::from(ParsedDocumentInner {
-        ast: document,
-        executable: Arc::new(executable),
-        ..Default::default()
-    });
+    let schema = crate::spec::Schema::parse(include_str!("./testdata/supergraph.graphql")).unwrap();
```

Comment:
```suggestion
    let schema = crate::spec::Schema::parse_test(include_str!("./testdata/supergraph.graphql"), Default::default()).unwrap();
```
We should use `parse_test` everywhere in tests to ensure that the queries are pointing to the API schema rather than the supergraph schema, as they do in a running router